pom.xml

```diff
@@ -150,9 +150,22 @@
         <zstdjni.version>1.5.7-3</zstdjni.version>
 
         <powsyblmathnative.version>1.4.0</powsyblmathnative.version>
+
+        <!-- Other variables -->
+        <enable-dynamic-agent-loading.argLine/>
     </properties>
 
     <profiles>
+        <profile>
+            <id>java21+</id>
+            <activation>
+                <jdk>[21,)</jdk>
+            </activation>
+            <properties>
+                <!-- In Java 21+, we have to add a parameter to allow dynamic agent loading (cf. powsybl-parent) -->
+                <enable-dynamic-agent-loading.argLine>-XX:+EnableDynamicAgentLoading</enable-dynamic-agent-loading.argLine>
+            </properties>
+        </profile>
```

Comment:
Can't we directly set the property, instead of setting it in a profile? We will always be in java21+ (it won't compile with lower versions).